test.sh

```diff
@@ -1,28 +1,26 @@
 #!/usr/bin/env bash
 
 set -x
-set -e
+#set -e
```

Comment:
`-e` will make the script exit if any of the checks within the `for` loop fails. I want _all_ the checks for all the specified versions to run before we declare failure—the fix might be working in one version of Cucumber but not the other.

The `FAILURE` variable will take care of returning a non-zero exit code in case of at least one test failure. So I’d like to not to re-add it, but remove it altogether.